correlation_rules/github_repo_cloned_followed_by_aws_login.yml

```diff
@@ -0,0 +1,26 @@
+AnalysisType: correlation_rule
+RuleID: "GitHub.Repo.Cloned.Followed.By.AWS.Login"
```

Comment:
Style question, is this form the standard that the Threat Research team has settled on for IDs? I'm more used to just seeing one or two periods to separate the logical components, i.e. `GitHub.RepoClonedFollowedByAWSLogin` or something like that.

Reply:
yep, we use dots https://github.com/panther-labs/panther-analysis/tree/release/rules/mongodb_rules